README.md

```diff
@@ -50,7 +50,31 @@ sim-cli sim.json
 The simulator requires access details for a set of `nodes` that you 
 have permission to execute commands on. Note that the current version 
 of the simulator uses keysend to execute payments, which must be 
-enabled in LND using `--accept-keysend`.
+enabled in LND using `--accept-keysend` (for CLN node it is enabled by default).
+
+The required access details will depend on the node implementation. For LND, the following
+information is required:
+
+```
+{
+  "id": <node_id>,
+  "address": <ip_and_port_or_domain_and_port>,
```

Comment:
nit: include `https://`

Reply:
Should we just move this line up? https://github.com/bitcoin-dev-project/sim-ln/blob/76c8ced0724a8a6c7989e47fef03e141b24e8221/README.md?plain=1#L113

Reply:
Works for me! But I'd still hard code it in before to get as close as possible - `https://<ip:port or domain:port>`

Reply:
Ok, I'm going to open an issue so we don't require this, and we can do that in a low-prio followup